src/core/fonts.js

```diff
@@ -2279,13 +2208,13 @@ var Font = (function FontClosure() {
       }
 
       var dupFirstEntry = false;
```

Comment:
Unless I'm mistaken, this can be removed now since it's no longer used for anything.